openshift/cronjob-template.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  annotations:
+    description: "This is Nepthys CronJob Config"
+    openshift.io/display-name: "Nepthys CronJob"
+    tags: "thoth,ai-stacks,nepthys"
+    template.openshift.io/documentation-url: "https://github.com/thoth-station/nepthys/"
+    template.openshift.io/long-description: "This is Nepthys used for automatic package releases and updates."
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+    version: "1.0.0"
+  labels:
+    app: nepthys
+    template: nepthys-cronjob
+  name: nepthys-cronjob
+objects:
+  - apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      labels:
+        app: nepthys
+      name: nepthys
+    spec:
+      concurrencyPolicy: Forbid
+      failedJobsHistoryLimit: 1
+      jobTemplate:
+        spec:
+          template:
+            metadata:
+              labels:
+                app: nepthys
+            spec:
+              containers:
+                - image: nepthys-job
+                  name: nepthys
+                  resources:
+                    limits:
+                      cpu: "2"
+                      memory: 2Gi
+                    requests:
+                      cpu: "2"
+                      memory: 2Gi
```

Comment:
Is it require to have 2Gi?

Reply:
I thought as we will be cloning and running sphinx on many repos, 2gi felt as quick enough.
I understand it for documentation so, I can reduce it to 1gi or less!